src/daemon/visor.go

```diff
@@ -432,13 +432,12 @@ func (vs *Visor) EstimateBlockchainLength() uint64 {
 	return maxLen
 }
 
-// HeadBkSeq returns the head sequence
-func (vs *Visor) HeadBkSeq() uint64 {
-	var seq uint64
+// HeadBkSeq returns the head sequence, returns -1 if blockchain is empty
+func (vs *Visor) HeadBkSeq() (seq uint64) {
```

Comment:
avoid named return values

Reply:
Why avoid named return value here

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters

"Don't name result parameters just to avoid declaring a var inside the function; that trades off a minor implementation brevity at the cost of unnecessary API verbosity."

Reply:
Ah, ok